### PR TITLE
Pre-fill rows based on the previous month

### LINKED ISSUE #5: Manual input setup
We'll need a template or guide or something to make sure users upload that data in good structured format.

### PR DESCRIPTION
**Pull request by jacebrowning**
Closes #5

### CODE REVIEW

**Comment by jacebrowning** [comment]
@brondsem Please test this with and without tab/month filtering.

File: dwellingplace/models/_utils.py [hunk 1]
```diff
@@ -92,17 +95,29 @@ def save_xlsx(data: list, sheets: List[str], path: str):
...
             row += 1
 
+        # Prefill rows
+        if prefill and prefill_cache:
+            most_recent_date = max(prefill_cache.keys())
+            next_date = increment_month(most_recent_date)
+            log.info("Prefilling data for %s", next_date)
+            for key in prefill_cache[most_recent_date]:
+                worksheet.write(row, 0, key)
+                # TODO: don't specify formatting here, use `get_value`
```
Comment (brondsem) [inline dwellingplace/models/_utils.py:116]: TODO left in code

**Comment by brondsem** [comment]
Looks good except for the TODO

File: dwellingplace/models/_utils.py (hunk 1 above)
Comment (jacebrowning) [reply dwellingplace/models/_utils.py:116]: Yeah, it needs to be cleaned up if we have time, but I'm going to merge this now.

**Pull request merged**